apiary_v2.apib

```diff
@@ -907,3 +907,363 @@ values for all roster fields in the template roster design is provided:
                 "original_request": "Not able to parse the request: `Unsupported media type \"type sent\" in request.`"
             }
 
+
+# Group Ecommerce Orders API
+
+The Ecommerce Orders API is accessed via your `https://partner-subdomain.embed.unmade.com/` URL. All requests require 
+your authorization token which is passed as an HTTP request header.
+
+## Order States
+
+As an order progresses through its lifecycle it will adopt one of a number of order states. These states are as:
+follows:
+
+    + `new`: This order has been received by Unmade, but is not in production yet
+    + `submitting`: This order is being sent to the factory for production
+    + `submitted`: This order has been sent to the factory for production
```

Comment:
Not important yet ... maybe to be reviewed if we want to rename them to something that makes more sense if possible?